src/Cygnus/DrushExport/AbstractExport.php

```diff
@@ -3,7 +3,7 @@
 namespace Cygnus\DrushExport;
 
 use InvalidArgumentException;
-use MongoClient;
+use \MongoClient;
```

Comment:
There's no need for this use if you're explicitly calling `new \MongoClient` below